rfc-008-server-naming-convention.md

```diff
@@ -0,0 +1,55 @@
+# Server naming convention
```

Comment:
Are we talking about `hostname`?

Reply:
I guess I wasn't specific at all here, but maybe that's a good thing so this can be adopted as and when and where needed. My immediate use is for Datadog server naming. Though this convention could be applied to DNS/hostnames too, and anything else.